ajax/libs/datatables.net-fixedheader-bs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "datatables.net-fixedheader-bs",
+  "version": "3.1.2",
+  "description": "FixedHeader for DataTables with styling for [Bootstrap](https://getbootstrap.com/)",
+  "filename": "css/fixedHeader.bootstrap.min.css",
+  "keywords": [
+    "fixed headers",
+    "sticky",
+    "DataTables",
+    "jQuery",
+    "table",
+    "Bootstrap"
+  ],
+  "homepage": "https://datatables.net/extensions/fixedheader",
```

Comment:
Would README.md in git repo be better? I can't find info about this styling library in the current link.